src/Hosting/StaticWebAssets/src/Microsoft.AspNetCore.StaticWebAssets.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core infrastructure for static web assets.</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
```suggestion
```

We should try to eliminate this NoWarn. Docs are a good thing and we should know where we have undocumented API. For all new API, I recommend removing this suppression.